internal/settings/settings.go

```diff
@@ -549,6 +532,22 @@ func setMemoryLimit(memTotalSizeMiB int) (int, error) {
 	return memLimit, nil
 }
 
+// Validate and set GOMEMLIMIT
```

Comment:
`GOMEMLIMIT` is already validated by the runtime, if it is invalid the go executable won't start (unless it is an old version that didn't support `GOMEMLIMIT`).

Reply:
good catch but we have more fundamental issue. More on stand up. for now, I am closing the PR.